yb-voyager/cmd/import.go

```diff
@@ -344,10 +350,15 @@ func validateImportObjectsFlag(importObjectsString string, flagName string) {
 }
 
 func checkOrSetDefaultTargetSSLMode() {
+	tconf.SSLMode = strings.ToLower(tconf.SSLMode) // normalize before comparing
+
 	if tconf.SSLMode == "" {
-		tconf.SSLMode = "prefer"
-	} else if tconf.SSLMode != "disable" && tconf.SSLMode != "prefer" && tconf.SSLMode != "require" && tconf.SSLMode != "verify-ca" && tconf.SSLMode != "verify-full" {
-		utils.ErrExit("Invalid sslmode %q. Required one of [disable, allow, prefer, require, verify-ca, verify-full]", tconf.SSLMode)
+		tconf.SSLMode = constants.PREFER
+		return
+	}
+
+	if !slices.Contains(supportedSSLModesOnTargetForImport, tconf.SSLMode) {
```

Comment:
This seems to be run in all import cases (import-data-to-target, import-data-to-source-replica, import-data-to-source), so the supportedSSLmodes should be determined based on the command (since we're defining `supportedSSLModesOnTargetForImport` and `supportedSSLModesOnSourceOrSourceReplica`

Reply:
updated. please check